extensions/wikia/CachedRedirects/CachedRedirects_setup.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+$wgExtensionCredits['other'][] = array(
```

Comment:
Given all the cache is used inside the FilePage extension in the WikiaFilePage class, I think it would be better to contain all the file page improvement logic in that one extension, rather than having the cache entries used and set by one extension and purged by a different extension.

Reply:
ok